src/cms/views/linkcheck/linkcheck.py

```diff
@@ -0,0 +1,106 @@
+from django.db.models import Q, OuterRef, Subquery
+from django.urls import reverse
+from django.views.generic import ListView
+from django.views.generic.base import RedirectView
+
+from linkcheck.models import Link
+from backend.settings import PER_PAGE
+
+from ...models.pages.page_translation import PageTranslation
+from ...models.events.event_translation import EventTranslation
+from ...models.pois.poi_translation import POITranslation
+
+
+# pylint: disable=too-many-ancestors
+class LinkListView(ListView):
```

Comment:
Finally somebody makes use of the Django inbuilt views :tada: :tada:
I really like it!